tests/tests_integration/test_api_client.py

```diff
@@ -392,7 +392,7 @@ def test_retrieve_multiple__ordering_matches_input(
     ) -> None:
         # Between SDK version 7.0.0 and 7.33.1, ordering of results was broken when >> 1k elements
         # was requested (meaning multiple requests were used):
-        event_ids = cognite_client.events.list(limit=1000).as_ids()
+        event_ids = cognite_client.events.list(limit=10).as_ids()
```

Comment:
I think this value should be 2-3x (at least) of the retrieve limit to not break what the test is for. Hey from the rainy cabin 😅

Reply:
Ack. Problem is that it keeps listing out stuff that is getting deleted by the time the byids call is made, but we can sort by descending createdTime to only get events which are not likely to be part of a current test run.